data/sql_updates/candidate_fulltext.sql

```diff
@@ -15,9 +15,9 @@ with nicknames as (
         link.cmte_id = totals.committee_id and
         link.fec_election_yr = totals.cycle
     where
-        cmte_dsgn in ('P', 'A') and (
-            substr(cand_id, 1, 1) = cmte_tp or
-            cmte_tp not in ('P', 'S', 'H')
+        totals.cmte_dsgn in ('P', 'A') and (
```

Comment:
I don't think we want to update to fulltext search. `ofec_candidate_fulltext_mv` is what powers the typeahead and we want to keep it as lean and fast as we can.

Reply:
I had to make this change, to fix the "ambiguous column" error that i ran into. since "disclosure.cand_cmte_linkage" table and the mv "ofec_totals_combined_mv_tmp" are now referring to the same columns in candidate_fulltext.sql.

Reply:
i had a sql code review with @PaulClark2 and @fecjjeng on this last week , whether to fetch the columns for ofec_totals_combined_mv_tmp or from the cand_cmte_linkage. Paul suggested to fetch from the ofec_totals_combined_mv_tmp.
cc : @PaulClark2 & @fecjjeng

Reply:
Thanks for the context. That makes sense.